sp/src/game/server/episodic/grenade_hopwire.cpp

```diff
@@ -2438,7 +2440,11 @@ void CGrenadeHopwire::DelayThink()
 
 	if( !m_bHasWarnedAI && gpGlobals->curtime >= m_flWarnAITime )
 	{
+//#ifndef EZ2
 		CSoundEnt::InsertSound ( SOUND_DANGER, GetAbsOrigin(), 400, 1.5, this );
+//#else
+//		CSoundEnt::InsertSound( SOUND_DANGER, GetAbsOrigin(), 400, m_flDetonateTime - gpGlobals->curtime, this );
+//#endif
```

Comment:
Why is this being changed back?

Reply:
That's a good question - I was trying to change the way that the timing works for the initial warning sound to make it always run until the grenade goes off. I commented out the change because I don't think it was necessary, and it also affects the Xen grenade. I don't want to make changes to the Xen grenade that could affect how it works unless they're necessary.